moto/resourcegroupstaggingapi/models.py

```diff
@@ -70,6 +71,10 @@ def __init__(self, region_name: str, account_id: str):
     def appsync_backend(self) -> AppSyncBackend:
         return appsync_backends[self.account_id][self.region_name]
 
+    @property
+    def athena_backend(self) -> AthenaBackend:
+        return athena_backends[self.account_id][self.region_name]
+
```

Comment:
You have to use this property or get rid of it, otherwise it's uncovered code.  (I would suggest using it on line 320.)